docs/zh_cn/advanced_guides/datasets/kitti_det.md

```diff
@@ -78,39 +76,42 @@ kitti
-    - 困难度：kitti 官方所定义的困难度，包括 简单，适中，困难
-    - 组别标识符：用于多部件的目标
-      }
-  - (optional) info\['calib'\]: {
-    - P0：校对后的 camera0 投影矩阵，是一个 3x4 数组
-    - P1：校对后的 camera1 投影矩阵，是一个 3x4 数组
-    - P2：校对后的 camera2 投影矩阵，是一个 3x4 数组
-    - P3：校对后的 camera3 投影矩阵，是一个 3x4 数组
-    - R0_rect：校准旋转矩阵，是一个 4x4 数组
-    - Tr_velo_to_cam：从 Velodyne 坐标到相机坐标的变换矩阵，是一个 4x4 数组
-    - Tr_imu_to_velo：从 IMU 坐标到 Velodyne 坐标的变换矩阵，是一个 4x4 数组
-      }
-  - (optional) info\['image'\]:{'image_idx': idx, 'image_path': image_path, 'image_shape', image_shape}.
-
-**注意**：其中的 info\['annos'\] 中的数据均位于相机参考坐标系中，更多的细节请参考[此处](http://www.cvlibs.net/publications/Geiger2013IJRR.pdf)。
-
-获取 kitti_infos_xxx.pkl 和 kitti_infos_xxx_mono3d.coco.json 的核心函数分别为 [get_kitti_image_info](https://github.com/open-mmlab/mmdetection3d/blob/7873c8f62b99314f35079f369d1dab8d63f8a3ce/tools/data_converter/kitti_data_utils.py#L140) 和 [get_2d_boxes](https://github.com/open-mmlab/mmdetection3d/blob/7873c8f62b99314f35079f369d1dab8d63f8a3ce/tools/data_converter/kitti_converter.py#L378).
+- `kitti_gt_database/xxxxx.bin`：训练数据集中包含在 3D 标注框中的点云数据
+- `kitti_infos_train.pkl`：训练数据集，其中每一帧的信息包含 `metainfo` 和 `data_list` 两个键值。`metainfo` 是一个字典，它包含数据集的基本信息，例如 `CLASSES` 和 `version`。`data_list` 是一个列表，它包含了所有需要的数据信息，每一个元素是一个样本的详细信息字典。详细信息如下所示：
+  - info\['sample_idx'\]：样本在整个数据集的索引。
+  - info\['images'\]：多个相机捕获的图像信息。是一个字典，包含 5 个键值：`CAM0`, `CAM1`, `CAM2`, `CAM3`, `R0_rect`。
+    - info\['images'\]\['R0_rect'\]：校准旋转矩阵，是一个 4x4 数组。
+    - info\['images'\]\['CAM2'\]：包含 `CAM2` 相机传感器的信息。
+      - info\['images'\]\['CAM2'\]\['img_path'\]：图像文件的路径。
+      - info\['images'\]\['CAM2'\]\['height'\]：图像的高。
+      - info\['images'\]\['CAM2'\]\['width'\]：图像的宽。
+      - info\['images'\]\['CAM2'\]\['cam2img'\]：相机到图像的变换矩阵，是一个 4x4 数组。
+      - info\['images'\]\['CAM2'\]\['lidar2cam'\]：激光雷达到相机的变换矩阵，是一个 4x4 数组。
+      - info\['images'\]\['CAM2'\]\['lidar2img'\]：激光雷达到图像的变换矩阵，是一个 4x4 数组。
+    - info\['lidar_points'\]：激光雷达捕获的点云信息。是一个字典，包含了激光雷达点云帧的信息。
+      - info\['lidar_points'\]\['lidar_path'\]：激光雷达点云数据的文件路径。
+      - info\['lidar_points'\]\['num_pts_feats'\]：每个点的特征数。
+      - info\['lidar_points'\]\['Tr_velo_to_cam'\]：Velodyne 坐标到相机坐标的变换矩阵，是一个 4x4 数组。
+      - info\['lidar_points'\]\['Tr_imu_to_velo'\]：IMU 坐标到 Velodyne 坐标的变换矩阵，是一个 4x4 数组。
+    - info\['instances'\]：目标检测任务所需要的。是一个列表，每个元素是包含了实例信息的字典。每个字典与该帧的一个实例标注相关。
+      - info\['instances'\]\['bbox'\]：长度为 4 的列表，以 (x1, y1, x2, y2) 的顺序表示实例的 2D 边界框。
+      - info\['instances'\]\['bbox_3d'\]：长度为 7 的列表，以 (x, y, z, w, h, l, yaw) 的顺序表示实例的 3D 边界框。
+      - info\['instances'\]\['bbox_label'\]：是一个整数，表示实例的 2D 标签，-1 代表忽略。
+      - info\['instances'\]\['bbox_label_3d'\]：是一个整数，表示实例的 3D 标签，-1 代表忽略。
+      - info\['instances'\]\['depth'\]：3D 边界框投影到相关图像平面的中心点的深度。
+      - info\['instances'\]\['num_lidar_pts'\]：3D 边界框内的激光雷达点数。
+      - info\['instances'\]\['center_2d'\]：3D 边界框投影的 2D 中心。
+      - info\['instances'\]\['difficulty'\]：Kitti 官方定义的困难度，包括简单、适中、困难。
+      - info\['instances'\]\['truncated'\]：实例框是否截断。
```

Comment:
EN: Float from 0 (non-truncated) to 1 (truncated), where truncated refers to the object leaving image boundaries
CN: .....